### PR TITLE
docs: add CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by timbray**
Closes: #18
Signed-off-by: Tim Bray <tbray@textuality.com>

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+
+## Basics
+
+Quamina is hosted in this GitHub repository 
+at `github.com/timbray/quamina` and welcomes 
+contributions.
+
+Typically, the first step in making a change is to 
+raise an Issue to allow for discussion of the idea. 
+This is important becuase possibly Quamina already
```
Comment (embano1) [inline CONTRIBUTING.md:11]: nit: typo because

**Comment by embano1** [comment]
One nit, LGTM then 👍

**Comment by embano1** [comment]
I'll work on the changelog stuff in the next days.

File: CONTRIBUTING.md (hunk 1 above)
Comment (timbray) [reply CONTRIBUTING.md]: Oops, fixed

**Comment by timbray** [comment]
If it's good enough for @embano1 it's probably good enough for v1.0. Merging.

**Pull request merged**